contrib/gitea-pull-checkout

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+user=$(echo $1 | cut -d ':' -f 1)
+branch=$(echo $1 | cut -d ':' -f 2)
+
+repo=$2
+[ -z "$repo" ] && repo=gitea
+
+git fetch "git@github.com:$user/$repo.git" "$branch:$branch"
+git checkout "$branch"
```

Comment:
I recommend inserting a new argument after `$1`: `PR number and then calling
```suggestion
git fetch "git@github.com:$user/$repo.git" "$branch:$2-$branch"
git checkout "$2-$branch"
```
that makes it a lot easier to keep track which branches belong to PRs.

Reply:
well $2 is mostly "" so we should use $user

Reply:
No, what I meant here is what I described in https://github.com/go-gitea/gitea/pull/25773#discussion_r1257345742 (now without typo)

Reply:
ah well yes if applyed ... I think the less args you need the better